src/css/base.css

```diff
@@ -6,13 +6,16 @@
   font-weight: 400;
 
   color: #242424;
-  background-color: rgba(255, 255, 255, 0.87);
-
+  /* background-color: rgba(255, 255, 255, 0.87); */
+  background-color: #f4f4f4;
   font-synthesis: none;
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   -webkit-text-size-adjust: 100%;
+  --color-black: #242424;
+  --color-white: #f4f4f4;
```

Comment:
you need to put this variable to the root container, then it can be used  in background-color: #f4f4f4;